apps/imb-mpi/ringpingpong.pbs

```diff
@@ -2,24 +2,31 @@
 
 # setup Intel MPI environment for Infiniband
 source /etc/profile # so we can load modules
-module load mpi/impi_2018.4.274
+module load mpi/impi
 source $MPI_BIN/mpivars.sh
 
 hostlist=$(pwd)/hosts.$PBS_JOBID
 
 sort -u $PBS_NODEFILE > $hostlist
 
+export I_MPI_FABRICS="shm:dapl"
```

Comment:
this won't work with later versions of intel mpi (and you dropped the version from the module load)

Reply:
yes I need to change this based on IMPI version